tools/generate_react_test_bundle.sh

```diff
@@ -4,6 +4,6 @@ then
     echo "You must run this tool from the root directory of your repo clone"
 else
     pushd libs/SalesforceReact
-    react-native bundle --platform android --dev true --entry-file node_modules/react-native-force/test/alltests.js --bundle-output ../test/SalesforceReactTest/assets/index.android.bundle
+    node_modules/.bin/react-native bundle --platform android --dev true --entry-file node_modules/react-native-force/test/alltests.js --bundle-output ../test/SalesforceReactTest/assets/index.android.bundle
```

Comment:
@wmathurin I updated this to use the locally installed `react-native` CLI instead of the global installation, since `install.sh` installs the correct version locally anyway.